File: src/main/java/group/yunxin/service/QuesService.java
package group.yunxin.service;

import java.util.List;

import group.yunxin.pojo.TbQues;
import group.yunxin.vo.PageResult;

/**
 * 服务层接口
 * 
 * @author Administrator
 *
 */
public interface QuesService
{

	/**
	 * 返回全部列表
	 * 
	 * @return
	 */
	public List<TbQues> findAll();

	/**
	 * 返回分页列表
	 * 
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);

	/**
	 * 增加
	 */
	public void add(TbQues ques);

	/**
	 * 修改
	 */
	public void update(TbQues ques);

	/**
	 * 根据ID获取实体
	 * 
	 * @param id
	 * @return
	 */
	public TbQues findOne(Long id);

	/**
	 * 批量删除
	 * 
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * 
	 * @param pageNum  当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbQues ques, int pageNum, int pageSize);

}

File: src/main/resources/static/js/service/quesService.js
//服务层
app.service('quesService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('/ques/findAll');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('/ques/findPage?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('/ques/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('/ques/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('/ques/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('/ques/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('/ques/search?page='+page+"&rows="+rows, searchEntity);
	}    	
});
